src/core/Config.cpp

```diff
@@ -102,10 +102,11 @@ void Config::init(const QString& fileName)
     m_defaults.insert("MinimizeOnCopy", false);
     m_defaults.insert("UseGroupIconOnEntryCreation", false);
     m_defaults.insert("AutoTypeEntryTitleMatch", true);
+    m_defaults.insert("UseGroupIconOnEntryCreation", true);
     m_defaults.insert("security/clearclipboard", true);
     m_defaults.insert("security/clearclipboardtimeout", 10);
     m_defaults.insert("security/lockdatabaseidle", false);
-    m_defaults.insert("security/lockdatabaseidlesec", 10);
+    m_defaults.insert("security/lockdatabaseidlesec", 240);
```

Comment:
Yes. 10 seconds is bullshit. it's a good timeout for clearing the clipboard, but locking the after 10 seconds is utter garbage. If you want instant lock, enable "Lock when minimizing" and then just send the window to the background once you're done with it. In any other case you probably need some time for whatever you're doing. And if you're logging into all your services in the morning, you really don't want to enter your master key for each and every one of them, because the DB locked itself in the meantime.